### ISSUE TITLE
Inicio do programa

### ISSUE THREAD

**Issue by jwcfj** [jwcfj/calculadorapolonesa#1]
Primeiramente deveríamos nos preocupar em formar uma entrada e transforma-la em lista.

**Comment by camila-ito** [comment]
Sim, nós poderíamos no inicio com intuito de teste colocar um vetor fixo e para formação da lista se utilizar de funções como malloc
